terraform/.terraform.lock.hcl

```diff
@@ -0,0 +1,19 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
I think this is related to your remote state and shouldn't be in the repo (maybe we should .gitignore it too)

Reply:
Or is this more like a `go.sum` 🤔

Reply:
https://www.terraform.io/docs/configuration/dependency-lock.html#lock-file-location
> You should include this file in your version control repository so that you can discuss potential changes to your external dependencies via code review, just as you would discuss potential changes to your configuration itself.

👍